zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1220,6 +1220,58 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /decision-requirements/{decisionRequirementsKey}/xml:
+    get:
+      tags:
+        - Decision requirements
+      summary: Get decision requirements XML (alpha)
```

Comment:
I think we need a period at the end.

Reply:
Actually Technical Writers say - no period for summary FYI @ralfpuchert 
<img width="566" alt="image" src="https://github.com/user-attachments/assets/0859476a-9ad3-4a99-b644-2a9e54601aba">